### PR TITLE
feat: [PIE-3374]: coverage improvement for files under pipelineSteps

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

> Added test cases for files under pipelineSteps

##### Jira Links:

https://harness.atlassian.net/browse/PIE-3374

##### Screenshots:

Earlier : 
image

Now :
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by dhruvUpadhyay07 — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx [hunk 1]
```diff
@@ -100,4 +100,79 @@ describe('Test K8sBlueGreenDeployStep', () => {
...
+          }
+        }}
+        type={StepType.HelmDeploy}
+        stepViewType={StepViewType.Edit}
+        onUpdate={onUpdate}
+        onChange={onChange}
+        ref={ref}
+      />
+    )
+    await ref.current?.submitForm()
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx:127]: The ideal test case for update would be modifying any property and then check if onChange was called with that modified properly. That would also check if changing values is working fine.

**Review by kapilkumawat86 — DISMISSED** [review]

File: src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx [hunk 2]
```diff
@@ -100,4 +100,79 @@ describe('Test K8sBlueGreenDeployStep', () => {
...
+        timeout: '<+input>',
+        type: StepType.HelmDeploy,
+        spec: {
+          timeout: '1s',
+          skipDryRun: false
+        }
+      },
+      viewType: StepViewType.TriggerForm
+    }
+    const response = new HelmDeploy().validateInputSet(data)
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx:175]: Better to do this real-time.
Submit the form and then check if `Value must be greater than or equal to \\"10s\\"` appears.

**Review by chetanvekariya-harness — COMMENTED** [review]
Added two comments and that applies to almost all test cases written in this PR.
Will be great if you can address these comments.

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx:187]: Resolved

File: src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/PipelineSteps/HelmDeploy/__tests__/HelmDeploy.test.tsx:132]: Resolved

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Pull request merged**